### DISCUSSION TITLE
How do i uninstall this plugin

### DISCUSSION THREAD

**Discussion by Josh65-2201** [sorayuki/obs-multi-rtmp#105] [Q&A] [answered]
It doesn't show up in the programs list to uninstall and I cant find an uninstaller file.

**Comment by frbelotto** [comment]
Did you find the answer? Same here!

**Reply by Josh65-2201** [reply]
No

**Comment by sorayuki** [comment, marked as answer]
Re-run the installer, it will ask if you want to do the uninstalling.
If it doesn't work, delete ```C:\ProgramData\obs-studio\plugins\obs-multi-rtmp``` folder manually.
Notice that ```ProgramData``` is a hidden folder.

**Reply by Josh65-2201** [reply]
Running the installer will just reinstall the plugin with no option for me to uninstall. But deleting the folder worked.

**Comment by SlavenIvanov** [comment]
How do I uninstall on OSX? Rerunning the installer does nothing. And there is no `obs-multi-rtmp` folder in the obs-studio/plugins folder.

**Reply by FrankSalad** [reply]
In Finder's Go menu, select `Go to Folder...`
In the popup type `~/Library/Application Support/obs-studio/plugins/`, to open the plugins folder, then move the `obs-multi-rtmp.plugin` and `obs-multi-rtmp.plugin.dSYM` to the trash.
